packages/sdk-auth/package.json

```diff
@@ -36,7 +36,7 @@
   },
   "dependencies": {
     "@commercetools/sdk-middleware-http": "^7.0.0",
-    "lodash.defaultsdeep": "^4.6.0",
+    "lodash.defaultsdeep": "^4.6.1",
```

Comment:
is this dependency still needed after the change to Object.assign?

Reply:
Just remove it 🙂